src/nyc_trees/apps/core/models.py

```diff
@@ -227,6 +229,18 @@ def blocks_mapped_count(self):
     def reservations_map_pdf_url(self):
         return url_if_cooked(self.reservations_map_pdf_filename)
 
+    def make_token(self):
+        return TimestampSigner().sign(self.username)
+
+    # Source: http://grokcode.com/819/one-click-unsubscribes-for-django-apps/
+    def is_valid_token(self, token):
+        try:
+            # Valid for 2 days
+            TimestampSigner().unsign(token, max_age=60 * 60 * 48)
```

Comment:
After thinking about this over lunch, I couldn't think of a good reason to bound the token with a TTL. Not something that needs to be addressed here though.